### ISSUE TITLE
PR for String parsing for TrueColor (closest match), and TrueColor for Hex.

### ISSUE THREAD

**Issue by EvanCarroll** [mackwic/colored#118]
It seems at `$work`, we use a fork of a fork of this project. Reviewing the code, I want to know if it's possible to get it included. It seems the original adds the ability to fall back if TrueColor isn't available. Our fork we use at work adds to that the ability to parse the colors from a string. This is all hidden behind a feature flag, so no added deps if not wanted. There are also tests.

Is this something the project would be interested in including?

The branch we use at work is https://github.com/jtk18/colored/tree/parse_truecolor
This is forked from https://github.com/spenserblack/colored

It seems spenserblack has an open PR up here. https://github.com/mackwic/colored/pull/90 But that PR has no feedback. Can more work be done to get that included upstream? If so, what can I do to help?

**Comment by hwittenborn** [comment]
Hey! I recently joined the team for this project, and I'd like to start getting a lot of these issues and PRs cleaned up. I'll give that PR a look soon so it can get merged in :)

**Comment by mackwic** [comment]
Hi, old maintainer here with not enough time to review and understand everything. Notably, this PR adds a new control environment (`$COLORTERM`) which I did not knew, and would like to understand better.

@EvanCarroll do you think that `$work` would be open for some sponsorship or this project, which would help us prioritize work on colored?

**Comment by EvanCarroll** [comment]
I don't think so, re sponsorship. Perhaps you should up to other maintainers that have more time. Seems @spenserblack may want to step up to the plate.

**Comment by spenserblack** [comment]
Actually, all my PRs here are years old, and I haven't used Rust in a while, so I'm not sure if I'm the best candidate :laughing:

But I'd be willing to at least help out with answering simple issues and reviewing some PRs. So maybe not "up to the plate," but "plate adjacent" :laughing:
